lib/lte_link_control/lte_lc.c

```diff
@@ -716,6 +766,22 @@ int lte_lc_init(void)
 
 void lte_lc_register_handler(lte_lc_evt_handler_t handler)
 {
+	static bool initialized;
+
+	if (!initialized) {
+		struct k_work_queue_config cfg = {
+			.name = "lte_lc_psm_get_work_q",
+		};
+
+		k_work_queue_start(
+			&lte_lc_psm_get_work_q,
+			lte_lc_psm_get_stack,
+			K_THREAD_STACK_SIZEOF(lte_lc_psm_get_stack),
+			LTE_LC_PSM_GET_PRIORITY,
+			&cfg);
```

Comment:
Maybe the work_q could be initialized using `SYS_INIT()` instead? You could also get rid of couple of comments above.

Reply:
Yep, makes sense.